app/src/debug/res/xml/shortcuts.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shortcuts xmlns:android="http://schemas.android.com/apk/res/android">
+    <shortcut
+        android:enabled="true"
+        android:icon="@drawable/ic_list_grey600_24dp"
+        android:shortcutId="queue"
+        android:shortcutShortLabel="@string/queue_label">
+        <intent
+            android:action="android.intent.action.VIEW"
+            android:targetClass="de.danoeh.antennapod.activity.MainActivity"
+            android:targetPackage="de.danoeh.antennapod.debug">
```

Comment:
Do you know another way to do this? I don't really like copying files with only little differences.

Reply:
@ByteHamster 

Thanks for your comments!

This article says, 

> xml - Static android shortcuts for multiple flavors? - Stack Overflow
> https://stackoverflow.com/questions/40361296/static-android-shortcuts-for-multiple-flavors

> the package name in the intent can not refer to a string resource and must be hardcoded in the xml.

ResourcePlaceholdersPlugin plugin can be used for this problem.

timfreiheit/ResourcePlaceholdersPlugin: use manifest placeholders in android resources
https://github.com/timfreiheit/ResourcePlaceholdersPlugin

789af2b uses this plugin, and app shortcut works well without copying shortcuts.xml.